### PR TITLE
SOC-1516 Create skeleton for notification service

### PR DESCRIPTION
**Pull request by jsutterfield**
Ticket: https://wikia-inc.atlassian.net/browse/SOC-1516

### CODE REVIEW

File: service/notification/src/main/java/com/wikia/services/notification/NotificationApplication.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  public void initialize(Bootstrap<NotificationConfiguration> bootstrap) {
+    super.initialize(bootstrap);
+
+    bootstrap.addBundle(SwaggerBundleProvider.<NotificationConfiguration>get());
+  }
+
+  @Override
+  public void run(NotificationConfiguration configuration, Environment environment)
+      throws Exception {
+    configurePublicCors(environment);
```
Comment (armonr) [inline service/notification/src/main/java/com/wikia/services/notification/NotificationApplication.java]: Please remove this per #571

File: service/notification/src/main/resources/banner.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+ _____
```
Comment (armonr) [inline service/notification/src/main/resources/banner.txt]: No banner love? :symbols:

**Comment by rafalkalinski** [comment]
LGTM

File: service/notification/src/main/java/com/wikia/services/notification/NotificationApplication.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+import com.theoryinpractise.halbuilder.api.RepresentationFactory;
+import com.theoryinpractise.halbuilder.jaxrs.JaxRsHalBuilderSupport;
+import com.theoryinpractise.halbuilder.standard.StandardRepresentationFactory;
+import io.dropwizard.setup.Bootstrap;
+import io.dropwizard.setup.Environment;
+
+public class NotificationApplication extends ConsulGuiceApplication<NotificationConfiguration> {
+
+  public static final RepresentationFactory
```
Comment (sqreek) [inline service/notification/src/main/java/com/wikia/services/notification/NotificationApplication.java]: We are not using RepresantationFactory pattern anymore.
Return Response class in as a response in your resources

File: service/notification/src/main/java/com/wikia/services/notification/NotificationApplication.java [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+import com.wikia.dropwizard.consul.ConsulGuiceApplication;
+import com.wikia.pandora.core.swagger.SwaggerBundleProvider;
+
+import com.theoryinpractise.halbuilder.api.RepresentationFactory;
+import com.theoryinpractise.halbuilder.jaxrs.JaxRsHalBuilderSupport;
+import com.theoryinpractise.halbuilder.standard.StandardRepresentationFactory;
+import io.dropwizard.setup.Bootstrap;
+import io.dropwizard.setup.Environment;
+
+public class NotificationApplication extends ConsulGuiceApplication<NotificationConfiguration> {
```
Comment (sqreek) [inline service/notification/src/main/java/com/wikia/services/notification/NotificationApplication.java]: Instead inherit from ConsulGuiceApplication add ConsuleBundle and PandoraGuiceBundle into environment

File: settings.gradle [hunk 5]
```diff
@@ -83,3 +83,4 @@ include 'service:user-permissions'
 include 'service:user-preference'
 include 'service:whoami'
 include 'workers:tcs-worker'
+include 'service:notification'
```
Comment (sqreek) [inline settings.gradle]: Please try to keep settings.gradle in group/alphabetic order

**Comment by sqreek** [comment]
Sorry for outdated template.
We will fix it as soon as possible

File: settings.gradle (hunk 5 above)
Comment (armonr) [inline settings.gradle]: @sqreek, I think this is also the tool just appending the new service to the end of the file

Comment (sqreek) [inline settings.gradle]: Yes it is.
But also I don't think we want to fix it.
(time(work on this feature) > time(sort this once a while))

Comment (armonr) [inline settings.gradle]: Yes, good point. I added a little note in [the wiki](https://github.com/Wikia/pandora/wiki/Service-Creation#create-new-service).

File: service/notification/src/main/resources/banner.txt (hunk 2 above)
Comment (jsutterfield) [inline service/notification/src/main/resources/banner.txt]: Oof! Terrible mistake, thanks for catching that :)

**Comment by armonr** [comment]
Thanks for the fixes. Tests are failing though.

**Comment by jsutterfield** [comment]
Good catch @armonr! Removed them per our discussion. We'll re-add similar tests when we start to implement the actual code.

**Comment by armonr** [comment]
:+1:

**Pull request merged**